pkg/vulnsrc/nvd/types.go

```diff
@@ -1,79 +1,156 @@
 package nvd
 
-type NVD struct {
-	CVEItems []Item `json:"CVE_Items"`
-}
-
-type Item struct {
-	Cve              Cve
-	Impact           Impact
-	LastModifiedDate string `json:"lastModifiedDate"`
-	PublishedDate    string `json:"publishedDate"`
-}
-
+// Cve is based on https://csrc.nist.gov/schema/nvd/api/2.0/cve_api_json_2.0.schema (see `cve_item`)
 type Cve struct {
-	Meta        Meta `json:"CVE_data_meta"`
-	References  References
-	Description Description
-	ProblemType ProblemType
+	ID                string          `json:"id"`
+	SourceIdentifier  string          `json:"sourceIdentifier,omitempty"`
+	Published         string          `json:"published"`
+	LastModified      string          `json:"lastModified"`
+	VulnStatus        string          `json:"vulnStatus,omitempty"`
+	EvaluatorComment  string          `json:"evaluatorComment,omitempty"`
+	EvaluatorSolution string          `json:"evaluatorSolution,omitempty"`
+	EvaluatorImpact   string          `json:"evaluatorImpact,omitempty"`
+	CisaExploitAdd    string          `json:"cisaExploitAdd,omitempty"`
+	CisaActionDue     string          `json:"cisaActionDue,omitempty"`
+	Descriptions      []LangString    `json:"descriptions"`
+	Metrics           Metrics         `json:"metrics,omitempty"`
+	Weaknesses        []Weakness      `json:"weaknesses,omitempty"`
+	Configurations    []Configuration `json:"configurations,omitempty"`
+	References        []Reference     `json:"references"`
+	VendorComments    []VendorComment `json:"vendorComments,omitempty"`
```

Comment:
Can we remove unneeded fields? Other structs as well.

Reply:
Removed. Please let me know if I did something wrong.

Reply:
Good point!
I checked your changes and don't see problem with that.